src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -377,7 +385,9 @@
                       Lines="$(PackageVersionsPropsContents)"
                       Overwrite="true" />
 
-    <Message Importance="High" Text="$(RepositoryName) using package version properties saved at $(CurrentSourceBuiltPackageVersionPropsPath) and $(PreviouslySourceBuiltPackageVersionPropsPath)" />
+    <Message Importance="High"
```

Comment:
While you are at it, would you mind removing the `Importance` metadata here (which downgrades it to medium)? This line is the only one that shows up in the minimal console output aside from the repo build/test exec task.

Reply:
Fixed in https://github.com/dotnet/sdk/pull/48327/commits/850f56cb106aecaae2b92c26a7bba1ac0d5e1a03